src/HelloAgain.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+function HelloAgain() {
+  return (
+    <div>
```

Comment:
Not a blocker by any means, but for better accessibility, you might consider using semantic HTML here as a `div` essentially has no meaning. I might suggest using `main` here in this case. You also could use a React Fragment if you ever come across a case where you need to wrap elements for some reason but they don't really fit with any of the semantic tags.